Content.Client/VendingMachines/VendingMachineBoundUserInterface.cs

```diff
@@ -42,7 +40,7 @@ protected override void Open()
             _menu.OnItemSelected += OnItemSelected;
             _menu.OnSearchChanged += OnSearchChanged;
 
-            _menu.Populate(_cachedInventory, _mod);
+            _menu.Populate(_cachedInventory, null, _mod);
```

Comment:
see previous comment. Having your nullable parameters at the end means you dont need to define null here